modules/firestore/README.md

```diff
@@ -0,0 +1,167 @@
+# Firestore
+
+This module allows to a fiewstore datatabase, fields, indexes and documents.
+
+## Examples
+
+### New database
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+}
+# tftest modules=1 resources=1 inventory=new-database.yaml
+```
+
+### New database with weekly backup
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+  backup_schedule = {
+    retention         = "86400s"
+    weekly_recurrenve = "MONDAY"
+  }
+}
+# tftest modules=1 resources=2 inventory=new-database-with-weekly-backup.yaml
+```
+
+### New database with document
+
+```hcl
+module "firestore" {
+  source     = "./fabric/modules/firestore"
+  project_id = "my-project"
+  database = {
+    name        = "my-database"
+    location_id = "nam5"
+    type        = "FIRESTORE_NATIVE"
+  }
+  documents = [{
```

Comment:
would it make sense to use a map here? for example, if any fields are dynamic this might trigger a dynamic cound error as there are no explicit keys

Reply:
done